bazel/check_deps/check_non_test_cc_deps.py

```diff
@@ -22,9 +22,7 @@
 from pathlib import Path
 
 runfiles = Path(os.environ["TEST_SRCDIR"])
-deps_path = (
-    runfiles / "carbon" / "bazel" / "check_deps" / "non_test_cc_deps.txt"
-)
+deps_path = runfiles / "_main" / "bazel" / "check_deps" / "non_test_cc_deps.txt"
```

Comment:
Should we also remove the `bazel-execroot` symlink, or update it to point to `_main`?

Reply:
Updated for _main